caddyhttp/markdown/markdown.go

```diff
@@ -43,6 +45,8 @@ type Markdown struct {
 
 	// The list of markdown configurations
 	Configs []*Config
+
+	BufPool *sync.Pool // docs: "A Pool must not be copied after first use."
```

Comment:
I dont think we need this comment, you can remove it.

Reply:
I copied this line (along with the comment) from the template implementation from #1649 
https://github.com/mholt/caddy/blob/2922d09bef3c504dde66bc12f7441668fcef6a20/caddyhttp/templates/templates.go#L132

I also don't think the comment needs to be there, but I'll defer to someone with more knowledge of the situation.